### DISCUSSION TITLE
Multithreaded Compositing

### DISCUSSION THREAD

**Discussion by SkelegonDK** [Zulko/moviepy#1870] [General]
Does anyone have any experience using multithreaded processing on compositing multiple videoclips at once?
something like this:

tasks
          composite[video, overlay],
          composite[video, overlay],
          composite[video, overlay],
          composite[video, overlay]
then 
      concat[newvideo,newvideo,newvideo,newvideo]

**Comment by keikoro** [comment]
Please report this as a proper issue instead of using GitHub's Discussions feature, thank you.
